plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -360,91 +359,54 @@ export class SessionRecordingIngester {
             })
         }
 
-        await runInstrumentedFunction({
```

Comment:
so my theory/guess/wondering is that `runInstrumentedFunction` is returning a promise which we asait but don't deal with

and so the promise hangs around like an unwanted guest at a funeral causing the high RAM

Reply:
this of course might have no impact at all on blobby's RAM usage